### PR TITLE
feat: Add support for embed file with AWS Bedrock Titan Model

### PR DESCRIPTION
**Pull request by longzhihun**
# Description
## Type of Change

Please delete options that are not relevant.
- [ ] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
1. Firstly, we need to setup AWS IAM User AK/SK to get the permission of Bedrock
2. Secondly, config the `titan-embed-text-v1` as the embedding model.
3. Thirdly, upload a file to knowledge base, if all right, the file can be embedded in a moment.

- [ ] TODO

# Suggested Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] I ran `dev/reformat`(backend) and `cd web && npx lint-staged`(frontend) to appease the lint gods
- [ ] `optional` I have made corresponding changes to the documentation 
- [ ] `optional` I have added tests that prove my fix is effective or that my feature works
- [ ] `optional` New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by crazywoola — APPROVED** [review]
LGTM

**Review by guchenhe — DISMISSED** [review]
LGTM! thanks for the PR, I've sent you an invite to join the discord contributor chat 🙏

**Comment by crazywoola** [comment]
Hello thank you for your contribution, you can add my wechat: crazyphage
I will invite you to our contributor's group.

File: api/core/model_runtime/model_providers/bedrock/text_embedding/_position.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+- amazon.titan-embed-text-v1
```
Comment (crazywoola) [inline api/core/model_runtime/model_providers/bedrock/text_embedding/_position.yaml]:
```suggestion
- amazon.titan-embed-text-v1

```

File: api/core/model_runtime/model_providers/bedrock/text_embedding/amazon.titan-embed-text-v1.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+model: amazon.titan-embed-text-v1
+model_type: text-embedding
+model_properties:
+  context_size: 8192
+pricing:
+  input: '0.0001'
+  unit: '0.001'
+  currency: USD
```
Comment (crazywoola) [inline api/core/model_runtime/model_providers/bedrock/text_embedding/amazon.titan-embed-text-v1.yaml]:
```suggestion
  currency: USD

```

**Review by crazywoola — DISMISSED** [review]

**Pull request merged**